### PR TITLE
Small edits to screening qs

### PR DESCRIPTION
**Pull request by NeilMcW**

### CODE REVIEW

File: app/views/v1/screening-questions/how-long.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+When did you get married?
 {% endblock %}
 
 {% block form %}
 <fieldset class="form-group">
   <legend class="visuallyhidden">Date of {{session.union_user}}</legend>
   <div class="form-date">
-    <label class="form-label-bold">Date you were married</label>
-    <label class="form-label">This must match the date on your marriage certificate</label>
+    <!-- <label class="form-label-bold">Date you were married</label> -->
```
Comment (anperez78) [inline app/views/v1/screening-questions/how-long.html]: comment to be removed

File: app/views/v1/screening-questions/respondent-address.html [hunk 2]
```diff
@@ -1,15 +1,14 @@
 {% extends "question.html" %}
 
 {% block question %}
-Do you have the Respondent’s address?
+Do you know your husband/wife's address?
```
Comment (anperez78) [inline app/views/v1/screening-questions/respondent-address.html]: 'husband/wife' to be replaced with {{ husbandOrWife() }}

File: app/views/v1/screening-questions/respondent-address.html [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 {% endblock %}
 
 {% block form %}
 
 <fieldset class="form-group inline">
-    <label class="form-label-bold text">
-      We will need to contact your husband or wife to confirm the details on this application form are correct.
-    </label>
+    <label class="form-label">
+Your husband/wife will need to be sent the divorce application    </label>
```
Comment (anperez78) [inline app/views/v1/screening-questions/respondent-address.html]: 'husband/wife' to be replaced with {{ husbandOrWife() }}

File: app/views/v1/screening-questions/respondent-address.html (hunk 2 above)
Comment (sophiedrake) [inline app/views/v1/screening-questions/respondent-address.html]: probably best to leave this how it is and not use `{{ husbandOrWife }}` as we haven't asked the questions required yet

File: app/views/v1/screening-questions/respondent-address.html (hunk 3 above)
Comment (sophiedrake) [inline app/views/v1/screening-questions/respondent-address.html]: same as above

**Comment by sophiedrake** [comment]
+1

**Pull request merged**